dbt/models/iasworld/schema/iasworld.dweldat.yml

```diff
@@ -916,31 +882,208 @@ sources:
+                  min_value: 1
+                  max_value: 40000
+                  additional_select_columns: *select-columns-with-class
+                  config:
+                    where: |
+                      CAST(taxyr AS int) BETWEEN {{ var('test_qc_year_start') }} AND {{ var('test_qc_year_end') }}
+                      AND cur = 'Y'
+                      AND deactivat IS NULL
+                      AND class = '211'
+                  meta:
+                    description: >
+                      sfla (Building Square Footage) should be between
+                      1 and 40000 for class 211 cards
+              - accepted_range:
+                  name: iasworld_dweldat_sfla_between_1_and_20000_for_class_212
                   min_value: 1
                   max_value: 20000
-                  additional_select_columns: *select-columns
+                  additional_select_columns: *select-columns-with-class
+                  config:
+                    where: |
+                      CAST(taxyr AS int) BETWEEN {{ var('test_qc_year_start') }} AND {{ var('test_qc_year_end') }}
+                      AND cur = 'Y'
+                      AND deactivat IS NULL
+                      AND class = '212'
+                  meta:
+                    description: >
+                      sfla (Building Square Footage) should be between
+                      1 and 20000 for class 212 cards
+              - accepted_range:
+                  name: iasworld_dweldat_sfla_between_1_and_10000_for_class_234
+                  min_value: 1
+                  max_value: 10000
+                  additional_select_columns: *select-columns-with-class
+                  config:
+                    where: |
+                      CAST(taxyr AS int) BETWEEN {{ var('test_qc_year_start') }} AND {{ var('test_qc_year_end') }}
+                      AND cur = 'Y'
+                      AND deactivat IS NULL
+                      AND class = '234'
+                  meta:
+                    description: >
+                      sfla (Building Square Footage) should be between
+                      1 and 10000 for class 234 cards
+              - accepted_range:
+                  name: iasworld_dweldat_sfla_between_2001_and_3800_for_class_278
+                  min_value: 2001
+                  max_value: 3800
+                  additional_select_columns: *select-columns-with-class
+                  config:
+                    where: |
+                      CAST(taxyr AS int) BETWEEN {{ var('test_qc_year_start') }} AND {{ var('test_qc_year_end') }}
+                      AND cur = 'Y'
+                      AND deactivat IS NULL
+                      AND class = '278'
+                  meta:
+                    description: >
+                      sfla (Building Square Footage) should be between
+                      2001 and 3800 for class 278 cards
+              - accepted_range:
```

Comment:
**suggestion:** We may want to create similar tests for `yrblt`. Probably worth opening another issue so as not to hold this one up any longer.

Reply:
I was thinking the same thing, issue open here: https://github.com/ccao-data/data-architecture/issues/398